kuma/wiki/jobs.py

```diff
@@ -136,7 +136,10 @@ class DocumentTagsJob(KumaJob):
     We invalidate this when a document is saved only.
     Longer lifetime as tags are rarely modified
     """
-    refresh_timeout = 180
+
+    # A wiki document can be updated with a short interval
+    # So task for refreshing tags should be created always
+    refresh_timeout = 0
```

Comment:
This is not what ``refresh_timeout`` is for. It is the timeout for how long to wait for an asynchronous refresh before triggering another job (see [docs](http://django-cacheback.readthedocs.io/en/latest/advanced.html?highlight=refresh_timeout).

If you believe that tags should be cached for 0 seconds, then tags are not appropriate as a cacheback job.